app/controllers/support_interface/data_exports_controller.rb

```diff
@@ -8,6 +8,18 @@ def index
       @data_exports = DataExport.includes(:initiator).order(id: :desc).page(params[:page] || 1).per(30)
     end
 
+    def directory
+      @export_types = DataExport::EXPORT_TYPES
+    end
+
+    def view_export_information
+      @data_export = DataExport::EXPORT_TYPES[params[:data_export_id].to_sym]
+    end
+
+    def view_download_history
+      @data_exports = DataExport.includes(:initiator).where(name: params[:data_export_id].humanize).order(id: :desc).page(params[:page] || 1).per(30)
```

Comment:
Maybe not for this PR, but it's a bit wonky to map back and forth from the name to the symbol here — would it be better to annotate DataExport records with the `type`?